### ISSUE TITLE
functiongraph not being rendered on certain conditions

### ISSUE THREAD

**Issue by Sjiep** [jsxgraph/jsxgraph#326]
We have bumped into a strange case where a functiongraph is not visible in the board, i.e. doesn't seem to be rendered. I have created a [minimal example](https://codesandbox.io/s/agitated-sea-k9lpk?file=/index.html) that reproduces the problem (you might have to reload the preview if the jsxboard is just white). The code snippet is the following:

```js
  var board = JXG.JSXGraph.initBoard('box', {boundingbox: [-1.5, 2, 3.5, -1], axis:true});
  var f1 = board.create('functiongraph', [function(x){return (x-0.5)**2+0.5;}, -1.5, 3.5], { fixed: true, strokeWidth: 2 })
  var f2 = board.create('functiongraph', [function(x){return (x-0)**2+0.5;}, -1.5, 3.5], { fixed: true, strokeWidth: 2 })
```
In the example, `f1` is not being rendered, however:
* if you move the bounding box by clicking the arrows the `f1` will be shown, up or down requires just one click, left and right a couple of clicks
* if you change the function graph to `function(x){return (x-0.6)**2+0.5;}` (change 0.5 to 0.6), it will also show up. Also `function(x){return (x-0.46)**2+0.99;}` doesn't show up and there are more of such positions that don't show.
* if you make the boundingbox of the board `[-5, 5, 5, -5]` it also shows up correctly

Any idea what is causing this behavior?

**Comment by alfredwassermann** [comment]
Thank you very much for pointing this out. Incidentally, it seems that this has been solved quite recently (fbd023373bdba99b5df49b55196683ae2ee649b1). With a recent nightly build your construction should work.

**Comment by Sjiep** [comment]
Thanks! I made build from the master branch and it fixed our issue :tada:
